qemu/tests/dump_guest_core.py

```diff
@@ -0,0 +1,101 @@
+import logging
+import os
+
+from avocado.utils import process
+
+from virttest import utils_package
+from virttest import utils_misc
+
+
+def run(test, params, env):
+    """
+    Test dump-guest-core, this case will:
+
+    1) Start VM with dump-guest-core option.
+    2) Check host env.
+    3) Trigger a core dump in host.
+    4) Use gdb to check core dump file.
+    5) If dump-guest-core=on, use crash to check vmcore file
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environmen.
+    """
+
+    def check_env():
+        """
+        Check if host kernel version is same with guest
+        """
+        guest_kernel_version = session.cmd("uname -r").strip()
+        logging.info("host kernel version is %s" % host_kernel_version)
+        logging.info("guest kernel version is %s" % guest_kernel_version)
+        if host_kernel_version != guest_kernel_version:
+            test.cancel("Please update your host and guest kernel "
```

Comment:
Could you please add guest kernel version and host kernel verison in the error message ? @leidwang since it's hassle for user to check the versions from the logs for every time. Thanks.